src/models/rfcProps.js

```diff
@@ -40,7 +40,11 @@ const getRFCProperties = () => {
 		 */
 		description: {
 			readableName: t('calendar', 'Description'),
-			placeholder: t('calendar', 'Add a description'),
+			placeholder: t('calendar', 'Add a description\n'
```

Comment:
From https://github.com/nextcloud/calendar/issues/6583#issue-2750460166

`Add a description …`

Reply:
@jancborchardt should there be `…` or not?

Reply:
I think the issue is that not all languages include the `...` when they translate it, but is this really an issue?

Reply:
I remember that there was a discussion about using `…` or not, but I don't remember the outcome. If Jan thinks this should be in then we add it, else we leave it. The `…` was definitely in the specification I have received from designers.